### PR TITLE
Update registry

### PR DESCRIPTION
**Pull request by Mrtenz**
This branch has a couple of PRs made by @Montoya, but grouped as one to avoid updating the registry a bunch of times.

### CODE REVIEW

File: src/registry.json [hunk 1]
```diff
@@ -411,6 +411,17 @@
         }
       }
     },
+    "npm:@forta-network/metamask-snap": {
+      "id": "npm:@forta-network/metamask-snap",
```
Comment (FrederikBolding) [inline src/registry.json:415]: `packageName` is not up to date in this `snap.manifest.json`. Would ask that they update before we add them.

File: src/registry.json [hunk 2]
```diff
@@ -432,6 +443,17 @@
           "checksum": "4DHXPu5lFAka46ohpDpyfwcSjT73s+nnZnO8PrVEr18="
         }
       }
+    },
+    "npm:@chainsafe/aleo-snap": {
+      "id": "npm:@chainsafe/aleo-snap",
+      "metadata": {
+        "name": "Aleo"
```
Comment (FrederikBolding) [inline src/registry.json:450]: `proposedName` is "Aleo snap"

File: src/registry.json (hunk 1 above)
Comment (Montoya) [reply src/registry.json:415]: OK I have asked them to update

File: src/registry.json (hunk 2 above)
Comment (Montoya) [reply src/registry.json:450]: I have asked the team to update

File: src/registry.json [hunk 3]
```diff
@@ -348,7 +348,7 @@
     "npm:mina-snap": {
       "id": "npm:mina-snap",
       "metadata": {
-        "name": "Mina"
+        "name": "MinaPortal"
```
Comment (FrederikBolding) [inline src/registry.json:351]: To be effective, this needs to be changed in the manifest

File: src/registry.json (hunk 1 above)
Comment (Montoya) [reply src/registry.json:415]: OK Forta has published v0.1.2 with the fix

File: src/registry.json (hunk 3 above)
Comment (Montoya) [reply src/registry.json:351]: Yes they will publish an update very soon

Comment (Montoya) [reply src/registry.json:340]: Update published.

File: src/registry.json (hunk 2 above)
Comment (Montoya) [reply src/registry.json]: Removing until it is updated

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**